### PR TITLE
[dependency] fix: upgrade the symbol-sdk for faucet

### PR DESCRIPTION
**Pull request by Wayonb**
problem: symbol-sdk 3.2.1 does not have a default import
solution: use named imports instead of default

### CODE REVIEW

File: faucet/backend/src/facade/nemFacade.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
 import { config } from '../config/index.js';
 import createNemClient from '../services/nemClient.js';
 import helper from '../utils/helper.js';
-import symbolSDK from 'symbol-sdk';
-
-const { PrivateKey, facade, nem } = symbolSDK;
+import { PrivateKey } from 'symbol-sdk';
+import { NemFacade, NetworkTimestamp } from 'symbol-sdk/nem'; /* eslint import/no-unresolved: [2, { ignore: ['\.nem$'] }] */
```
[the remark this answers is not in the document]
Comment (Wayonb) [reply faucet/backend/src/facade/nemFacade.js:5]: due to this bug ``https://github.com/import-js/eslint-plugin-import/issues/1810``

File: faucet/backend/test/utils/helper_spec.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import { config } from '../../src/config/index.js';
 import helper from '../../src/utils/helper.js';
 import { expect } from 'chai';
-import symbolSDK from 'symbol-sdk';
+import { PrivateKey } from 'symbol-sdk';
+// disable until https://github.com/import-js/eslint-plugin-import/issues/1810 is resolved
+/* eslint import/no-unresolved: [2, { ignore: ['^symbol-sdk/'] }] */
```
Comment (Jaguar0625) [inline faucet/backend/test/utils/helper_spec.js:6]: can this be removed?

File: faucet/frontend/utils/helper.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
-import symbolSDK from 'symbol-sdk';
+/* eslint-disable import/extensions */
```
Comment (Jaguar0625) [inline faucet/frontend/utils/helper.js:1]: is this still needed?

**Review by Jaguar0625 — APPROVED** [review]
👍

**Review by AnthonyLaw — COMMENTED** [review]
👍🏼

**Pull request merged**